src/CreatorToken.sol

```diff
@@ -107,10 +152,17 @@ contract CreatorToken is ERC721Royalty {
     if (_referrer != address(0)) _mintAndIncrement(_referrer);
   }
 
+  /// @notice Purchase a token.
+  /// @param _maxPayment The maximum amount of USDC the buyer is willing to pay.
+  /// @return _totalPrice The total amount of USDC paid for the creator token.
   function buy(uint256 _maxPayment) public returns (uint256 _totalPrice) {
     _totalPrice = buy(msg.sender, _maxPayment);
   }
 
+  /// @notice Purchase a token for another address.
```

Comment:
nit: recommend changing the words to say the token is "minted to another address"

Reply:
Changed in https://github.com/showtime-xyz/creator-tokens/commit/3c46721e1c9a0899f54630e2b73028cd5bef159c